src/create_wand_data.cpp

```diff
@@ -9,53 +10,68 @@
 #include "util/util.hpp"
 #include "wand_data.hpp"
 #include "wand_data_compressed.hpp"
-#include "wand_data_raw.hpp"
 #include "wand_data_range.hpp"
+#include "wand_data_raw.hpp"
 
 #include "CLI/CLI.hpp"
 
-int main(int argc, const char **argv) {
+int main(int argc, const char **argv)
+{
     using namespace pisa;
 
     float lambda = 0.0f;
```

Comment:
Let's use `std::optional<float> lambda{};` here instead. It works out of the box. The difference is that now it will be `std::nullopt` when no option is passed. Then it makes more sense when you check for that later.

Reply:
Yes, great point!